src/custom_types/message.py

```diff
@@ -0,0 +1,31 @@
+from asyncio import Future
+
+from prompt_toolkit.layout.containers import HSplit
+from prompt_toolkit.layout.dimension import D
+from prompt_toolkit.widgets import Button, Dialog, Label
+
+from custom_types.ui_types import PopUpDialog
+
+
+class MessageDialog(PopUpDialog):
+    """About tab dialog box"""
+
+    def __init__(self, title: str, text: str):
+        self.future = Future()
+
+        def set_done() -> None:
+            """Future object when done return None"""
```

Comment:
Can we make this docstring better? I don't really understand how this works.

Reply:
ok, it's quite similar to how `ScrollMenuDialog.set_done()` works.